src/libPMacc/include/nvidia/functors/Assign.hpp

```diff
@@ -22,6 +22,8 @@
 
 #pragma once
 
+#include "pmacc_types.hpp"
+
 namespace PMacc
 {
     namespace nvidia
```

Comment:
Following the [coding guideline](https://github.com/ComputationalRadiationPhysics/picongpu/wiki/Coding-Guide-Lines#1-spaces-and-indentation) there should be no indentation here (and all following namespaces).